src/main/scala/edu/berkeley/ce/rockslicing/BlockVTK.scala

```diff
@@ -36,18 +36,18 @@ object BlockVTK {
     // vertical line running through this center.
     // Check if points are on opposite sides of the center. Points left of center will be before points
     // right of the center
-    if ((pointA(0) - center(0) < -NumericUtils.EPSILON) && (pointB(0) - center(0) >= NumericUtils.EPSILON)) {
```

Comment:
Why did we get rid of the use of `EPSILON` here?

For example, if something is greater than `-EPSILON` but less than 0.0, then it's basically 0.0, and thus wouldn't we not want to treat it as a value less than 0.0? Doesn't that behavior get lost with these changes?